### PR TITLE
Vignettes update

### PR DESCRIPTION
**Pull request by PavelBal**
- Added class descriptions to the intro-vignette
- Added information about simulation tree
- Added Pk-analyses vignette

### CODE REVIEW

File: vignettes/introduction-ospsuite.Rmd [hunk 1]
```diff
@@ -51,17 +52,40 @@ resultOfAFunction <- object1$multiply(1,2)
 Important information about the object can be printed out by calling `print(object)`.
 
 The most important classes are:
-  : `Simulation` Representation of the simulation loaded from the *.pkml file.
-  : `SimulationSettings` #Short descriptions of the classes to be added here
+  : `Simulation`
```
Comment (PavelBal) [inline vignettes/introduction-ospsuite.Rmd:55]: I think an overview of the most important classes with their short descriptions is useful to advanced user.

File: vignettes/pk-analysis.Rmd [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
```
Comment (PavelBal) [inline vignettes/pk-analysis.Rmd:1]: I have really little experience with PK-Analysis, so just covered the basics. Maybe someone can add a section about user-defined PK-parameters.

**Comment by PavelBal** [comment]
The PK-Analysis vignettes is not complete, but let's keep it in a separate branch (or merge it and someone should expand later...) so I can cover other topics in separate branches.

File: vignettes/introduction-ospsuite.Rmd (hunk 1 above)
Comment (Yuri05) [reply vignettes/introduction-ospsuite.Rmd:55]: I think it's useful for _all_ users 😄

File: vignettes/pk-analysis.Rmd (hunk 2 above)
Comment (Yuri05) [reply vignettes/pk-analysis.Rmd:1]: Created an issue for this https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/384

**Pull request merged**